stable_baselines/logger.py

```diff
@@ -124,8 +124,10 @@ def __init__(self, filename):
     def writekvs(self, kvs):
         for key, value in sorted(kvs.items()):
             if hasattr(value, 'dtype'):
-                value = value.tolist()
-                kvs[key] = float(value)
+                if value.shape == ():
+                    kvs[key] = float(value.tolist())
```

Comment:
why not `kvs[key] = float(value)` ?

Reply:
I'm not sure. The existing code is first converting the value to a list and then float. I wasn't sure why so I left it, but I can remove the `tolist`